packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -713,7 +758,45 @@ class IosProject extends XcodeBasedProject {
     }
   }
 
-  Future<void> _regenerateFromTemplateIfNeeded() async {
+  Future<void> _updateLLDBIfNeeded() async {
+    if (globals.cache.isOlderThanToolsStamp(lldbInitFile) ||
+        globals.cache.isOlderThanToolsStamp(lldbHelperPythonFile)) {
+      if (isModule) {
+        // When building a module project for Add-to-App, provide instructions
+        // to manually add the LLDB Init File to their native Xcode project.
+        globals.logger.printWarning(
+          'Debugging Flutter on new iOS versions requires an LLDB Init File. '
+          'To ensure debug mode works, please complete one of the following:\n'
+          '  * Set LLDB Init File in your scheme via Xcode > Product > Scheme > '
+          'Edit Scheme for both Run and Test to the following: \n'
```

Comment:
I wonder if there's a way we could reword this sentence such that its order mirrors the flow the user needs to do: open xcode, open Run action, edit LLDB Init File, open Test action, edit LLDB Init File. Maybe something like this?

```suggestion
          '  * Open Xcode > Product > Scheme > Edit Scheme. For both the Run and Test actions, set LLDB Init File to: \n'
```